api/src/paths/publish/attachment/resubmit.ts

```diff
@@ -0,0 +1,172 @@
+          properties: {
+            projectId: {
+              type: 'number',
+              minimum: 1
+            },
+            fileName: {
+              type: 'string'
+            },
+            parentName: {
+              type: 'string'
+            },
+            formValues: {
+              required: ['full_name', 'email_address', 'phone_number', 'description'],
+              properties: {
+                full_name: {
+                  type: 'string'
+                },
+                email_address: {
+                  type: 'string'
+                },
+                phone_number: {
+                  type: 'string'
+                },
+                description: {
+                  type: 'string'
+                }
+              }
+            },
+            path: {
+              type: 'string'
+            }
+          }
+        }
+      }
+    }
+  },
+  responses: {
+    200: {
+      description: 'Resubmit OK',
+      content: {
+        'application/json': {
+          schema: {
+            title: 'email sent response',
+            type: 'boolean'
+          }
+        }
+      }
+    },
+    400: {
+      $ref: '#/components/responses/400'
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    403: {
+      $ref: '#/components/responses/401'
+    },
+    500: {
+      $ref: '#/components/responses/500'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+/**
+ * Publish project data to Biohub.
+ *
+ * @return {*}  {RequestHandler}
+ */
+export function resubmitAttachment(): RequestHandler {
+  return async (req, res) => {
+    const connection = getDBConnection(req['keycloak_token']);
+
+    const resubmitData = req.body;
+    const url = `${process.env.APP_HOST}/login?redirect=${encodeURIComponent(resubmitData.path)}`;
+    const hrefUrl = `[${resubmitData.parentName}](${url})`;
+
+    const message: IgcNotifyRequestRemovalMessage = {
+      subject: '',
+      header: '',
+      date: new Date().toLocaleString(),
+      file_name: resubmitData.fileName,
+      link: hrefUrl,
+      description: resubmitData.formValues.description,
+      full_name: resubmitData.formValues.full_name,
+      email: resubmitData.formValues.email_address,
+      phone: resubmitData.formValues.phone_number
+    };
+
+    const submitterMessage: IgcNotifyRequestRemovalMessage = {
+      ...message,
+      subject: 'Species Inventory Management System - Your Request to Remove or Resubmit Has Been Sent',
+      header: `Your request to remove or resubmit data has been sent.
+
+      A BioHub Administrator should be in contact with you shortly to discuss your request.`
+    };
+
+    const adminEmail = process.env.GCNOTIFY_ADMIN_EMAIL || '';
+    const adminMessage: IgcNotifyRequestRemovalMessage = {
+      ...message,
+      subject: 'Species Inventory Management System -  Request to Remove or Resubmit',
+      header: ''
+    };
```

Comment:
It would be more ideal if this content could be moved into the service level. 

In general, all we want to do in the endpoint is parse out request data and establish database connections, and handle the final response. Everything else is ideally in the service files.